src/stuf/stuf_zds/templates/stuf_zds/soap/updateZaak.xml

```diff
@@ -1,12 +1,12 @@
-<ZKN:zakLk01
-        xmlns:StUF="http://www.egem.nl/StUF/StUF0301"
+{% extends "stuf/soap_envelope.xml" %}{% load stuf %}
+{% block body %}<ZKN:zakLk01
         xmlns:ZKN="http://www.egem.nl/StUF/sector/zkn/0310"
```

Comment:
namespace mess - StUF-BG defines them on the root element, while StUF-ZDS defines them on the children? Why this inconsistency?

Reply:
Resolved via 0126688d73541bbe5ff157b1114d35cc71426dec